guard/test/support/factories/organization.ex

```diff
@@ -8,7 +8,8 @@ defmodule Support.Factories.Organization do
       open_source: false,
       description: "Test Organization Description",
       website: "https://example.com",
-      avatar_url: "https://example.com/avatar.png"
+      avatar_url: "https://example.com/avatar.png",
+      allowed_id_providers: "api_roken,oidc"
```

Comment:
Typo `api_roken -> api_token`, same in the other test file. It might be an issue for new tests in the future.

Reply:
Good catch, fixed